examples/go-union-type/index.ts

```diff
@@ -0,0 +1,23 @@
+import { GoGenerator } from '../../src';
+
+const generator = new GoGenerator();
+const jsonSchemaDraft7 = {
+  $schema: 'http://json-schema.org/draft-07/schema#',
+  type: 'object',
+  additionalProperties: {
```

Comment:
Do you mind changing the example so it outputs multiple Union types and then use the options for naming so we reflect those options you added?

Reply:
@Souvikns you never adapted the options to include all the `unionAnyModelName` and others. Right now users have no idea the options are there.